README.md

```diff
@@ -0,0 +1,14 @@
+# Hashflow: Sample Market Maker
+
+This example repo shows how to create a market maker which connects with the [**Hashflow**](hashflow.com) API via WebSocket.
+
+## Using
+Install all necessary requirements by running
+```
+$ yarn
+```
+
+Then, you can run the example file with 
+```
+$ yarn example
```

Comment:
same -- does it make sense to call it run?

Reply:
Yep, updated